examples/inverse/plot_psf_ctf_label_leakage.py

```diff
@@ -0,0 +1,209 @@
+"""
+===========================================================================
+Compute source leakage among labels and visualize it using a circular graph
+===========================================================================
+
+This example computes all-to-all pairwise leakage between 68 regions in
+source space based on MNE inverse solutions and a FreeSurfer cortical
+parcellation. Label-to-label leakage is estimated as the correlation among the
+labels' point-spread functions. It is visualized using a circular graph
+which is ordered based on the locations of the regions in the axial plane.
+"""
+# Authors: Olaf Hauk <olaf.hauk@mrc-cbu.cam.ac.uk>
+#          Martin Luessi <mluessi@nmr.mgh.harvard.edu>
+#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#          Nicolas P. Rougier (graph code borrowed from his matplotlib gallery)
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+from mne.minimum_norm import (read_inverse_operator,
+                              make_inverse_resolution_matrix,
+                              get_point_spread)
+
+from mne.viz import circular_layout, plot_connectivity_circle
+
+print(__doc__)
+
+###############################################################################
+# Load forward solution and inverse operator
+# ------------------------------------------
+#
+# We need a matching forward solution and inverse operator to compute
+# resolution matrices for different methods.
+
+data_path = sample.data_path()
+subjects_dir = data_path + '/subjects'
+fname_fwd = data_path + '/MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif'
+fname_inv = data_path + '/MEG/sample/sample_audvis-meg-oct-6-meg-fixed-inv.fif'
+
+# Read forward solution
+forward = mne.read_forward_solution(fname_fwd)
+# Convert to fixed source orientations
+mne.convert_forward_solution(
+    forward, surf_ori=True, force_fixed=True, copy=False)
+
+# Read inverse operator
+inverse_operator = read_inverse_operator(fname_inv)
+
+# Compute resolution matrices for MNE
+lambda2 = 1. / 3.**2
+method = 'MNE'
+rm_mne = make_inverse_resolution_matrix(forward, inverse_operator,
+                                        method=method, lambda2=lambda2)
```

Comment:
```suggestion
rm_mne = make_inverse_resolution_matrix(forward, inverse_operator,
                                        method='MNE', lambda2=lambda2)
```
(now that you're not doing sLORETA too, no need to parametrize the method arg)